src/main/scala/com/gu/newsletterlistcleanse/GetCleanseListLambda.scala

```diff
@@ -55,11 +57,12 @@ object GetCleanseListLambda {
 
     val results = for {
       campaignCutOff <- campaignCutOffDates
-      userIds = campaigns.fetchCampaignCleanseList(campaignCutOff).map(_.userId)
+      userIds = databaseOperations.fetchCampaignCleanseList(campaignCutOff).map(_.userId)
       cleanseList = CleanseList(
         campaignCutOff.newsletterName,
         userIds
       )
+      _ = logger.info(s"Found ${userIds.length} users to remove from ${campaignCutOff.newsletterName}")
```

Comment:
TODO: Add in a query to get the length of a campaign's user list in order to generate a proportional change

Reply:
I think the count is only useful for the circuit breaker, and once we've done the initial cleaning. So I'll consider the task is included in the circuit breaker ticket, but do let me know if you had anything else in mind